XiEditor/StatusBar.swift

```diff
@@ -112,32 +112,32 @@ class StatusBar: NSView {
         item.textColor = itemTextColor
         self.addSubview(item)
         currentItems[item.key] = item
-        checkItemsFitFor(windowWidth: self.superview!.frame.width)
         self.needsUpdateConstraints = true
+        checkItemsFitFor(windowWidth: self.superview!.frame.width)
```

Comment:
shouldn't `self.bounds.width` always suffice? our width should be the same as the superview's.